SingularityUI/app/rootComponent.jsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import classNames from 'classnames';
 
-// eslint-disable-next-line no-unused-vars react/no-multi-comp
```

Comment:
Forgot about this line. That `eslint-disable` was not necessary so I removed it when I thought the bugfix was in `rootComponent`.
Its deletion in this PR isn't harmful so I'm going to leave it deleted.